prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -274,57 +272,42 @@ func getReviewers(rc reviewersClient, ghc githubClient, log *logrus.Entry, autho
 		// record required reviewers if any
 		requiredReviewers.Insert(rc.RequiredReviewers(file.Filename).UnsortedList()...)
 
-		fileUnusedLeafs := rc.LeafReviewers(file.Filename).Difference(stage1).Difference(authorSet)
+		fileUnusedLeafs := layeredsets.NewString(rc.LeafReviewers(file.Filename).List()...).Difference(reviewers.Set()).Difference(authorSet)
 		if fileUnusedLeafs.Len() == 0 {
 			continue
 		}
 		leafReviewers = leafReviewers.Union(fileUnusedLeafs)
 		if r := findReviewer(ghc, log, useStatusAvailability, &busyReviewers, &fileUnusedLeafs); r != "" {
-			stage1.Insert(r)
+			reviewers.Insert(0, r)
 		}
 	}
 	// now ensure that we request review from at least minReviewers reviewers. Favor leaf reviewers.
-	unusedLeafs := leafReviewers.Difference(stage1)
-	for stage1.Len()+stage2.Len() < minReviewers && unusedLeafs.Len() > 0 {
+	unusedLeafs := leafReviewers.Difference(reviewers.Set())
+	for reviewers.Len() < minReviewers && unusedLeafs.Len() > 0 {
```

Comment:
What happened to the handling of stage 2 reviewers?

Reply:
This is the code inside the loop, stage2 is the layer 1 in `reviewers.Insert(1, r)`